client/components/domains/domain-suggestion/style.scss

```diff
@@ -346,7 +346,7 @@
 body.is-section-stepper,
 body.is-section-signup {
 	&:not(:has(.step-container-v2)) {
-		svg:not(.jetpack-logo, .social-buttons__button svg) {
+		svg:not(.step-container__jetpack-powered-logo, .social-buttons__button svg) {
```

Comment:
Using `.step-container__jetpack-powered-logo` instead of the internal `.jetpack-logo` class